### PR TITLE
Fix Wrenjs compilation

### LINKED ISSUE #4251: emcc compilation error
**Describe the Bug**
Fairly regularly for a week or two now I've been getting this error while compiling (master and develop branch alike). Cleansing has no effect, nor `git clean -dfx`. The error blocks the compilation, but if the same make command is executed after the error, the second time it compiles without complaining. Happens in the main PC, on the laptop, and in virtualbox.

```
# linking libwren.a
Error: Embedding "../../resources/wren/shaders/color_noise.frag" which is below the current directory "/home/daniel/webots_develop/src/wren". This is invalid since the current directory becomes the root that the generated code will see
emcc: error: '/home/daniel/webots_master/dependencies/emsdk/upstream/emscripten/tools/file_packager /home/daniel/webots_develop/resources/web/wwi/wrenjs.data --from-emcc --export-name=Module --preload ../../resources/wren/shaders' failed (returned 1)
make[1]: *** [/home/daniel/webots_develop/resources/Makefile.include:494: build/release/libwren.a] Error 1
make: *** [Makefile:112: webots_target] Error 2
```

### PR DESCRIPTION
**Pull request by BenjaminDeleze**
**Description**
Fix #4251

### CODE REVIEW

**Comment by BenjaminDeleze** [comment]
I will improve the Makefile to be sure that it will fail everytime if there is a problem in the futur

**Review by ad-daniel — DISMISSED** [review]
Thank you, appears to be working, can't reproduce it any longer

File: resources/Makefile.include [hunk 1]
```diff
@@ -539,18 +549,22 @@ $(BUILD_GOAL_DIR)/%.o:%.c++
...
 ### How to clean up the directory
 ###-----------------------------------------------------------------------------
 clean:
 	$(SILENT)rm -fr $(FILES_TO_REMOVE) $(BUILD_DIR) > /dev/null 2>&1 || :
 	$(SILENT)rm -d $(EMPTY_DIRECTORIES_TO_REMOVE) > /dev/null 2>&1 || :
-	$(SILENT)rm $(WRENJS_DIR)/wrenjs.js $(WRENJS_DIR)/wrenjs.data $(WRENJS_DIR)/wrenjs.wasm > /dev/null 2>&1 || :
-	$(SILENT)rm -r $(WRENJS_DIR)/images/post_processing > /dev/null 2>&1 || :
+  ifdef WREN
+	  $(SILENT)rm $(WEBOTS_HOME_PATH)/src/wren/wrenjs.js $(WEBOTS_HOME_PATH)/src/wren/wrenjs.data $(WEBOTS_HOME_PATH)/src/wren/wrenjs.wasm  $(WEBOTS_HOME_PATH)/src/wren/functions_to_export.txt > /dev/null 2>&1 || :
+	  $(SILENT)rm $(WRENJS_DIR)/wrenjs.js $(WRENJS_DIR)/wrenjs.data $(WRENJS_DIR)/wrenjs.wasm > /dev/null 2>&1 || :
```
Comment (omichel) [inline resources/Makefile.include:568]: Adding two spaces here looks strange, I would remove one of them:
```suggestion
	 $(SILENT)rm $(WEBOTS_HOME_PATH)/src/wren/wrenjs.js $(WEBOTS_HOME_PATH)/src/wren/wrenjs.data $(WEBOTS_HOME_PATH)/src/wren/wrenjs.wasm  $(WEBOTS_HOME_PATH)/src/wren/functions_to_export.txt > /dev/null 2>&1 || :
	 $(SILENT)rm $(WRENJS_DIR)/wrenjs.js $(WRENJS_DIR)/wrenjs.data $(WRENJS_DIR)/wrenjs.wasm > /dev/null 2>&1 || :
```

**Review by omichel — CHANGES_REQUESTED** [review]
It looks good to me.

**Review by omichel — APPROVED** [review]
Thank you.

**Pull request merged**